app/bundles/LeadBundle/Form/Type/TagType.php

```diff
@@ -57,8 +58,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
                 'label'         => 'mautic.lead.tags',
                 'class'         => 'MauticLeadBundle:Tag',
                 'query_builder' => function (EntityRepository $er) {
-                    return $er->createQueryBuilder('t')
-                        ->orderBy('t.tag', 'ASC');
+                    return $er->createQueryBuilder('t')->orderBy('t.tag', 'ASC');
```

Comment:
Better is to encapsulate this - like `$tagRepository->findAllTags()`. Function could be reused in other code.

Reply:
I was just removing MauticFactory dependency from this class. It wasn't my intention to rewrite it :)

Reply:
I see, but we need to clean a code when there is a chance ;) But this is not so important, it is very easy method.